scripts/auth0connections/auth0connections.py

```diff
@@ -29,26 +30,44 @@ def get_client():
 
 
 def render_local_connections(config):
+    '''Returns full connection dicts, from the connections listed in the config
+    file, and rendered using the template specified.
+    
+    Uses a template specified in the connection config
+    e.g. 'template_name: hmpps-auth' in the connection configuration
+         means use the template in: ./connection_templates/hmmps-auth/
+    '''
     connections = {}
-    connections_root = Path(__file__).cwd() / Path("connections")
-    connection_dirs = [entry for entry in connections_root.iterdir() if entry.is_dir()]
-
-    for connection in connection_dirs:
-        connection_name = connection.stem
-        scripts = connection.glob("*.js")
+    template_root = Path(__file__).cwd() / Path("connection_templates")
```

Comment:
I see the templates directory was renamed (seem to make sense if these are templates).
The README may need some tweaks as a result of the renaming.

Reply:
Good point. I've now updated the readme with this.